### ISSUE TITLE
Internet Explorer: sometimes points / labels not visible

### ISSUE THREAD

**Issue by olila** [CindyJS/CindyJS#697]
Under some circumstances, it seems that CindyJS does not display points / labels in Windows Internet Explorer.

I created a minimal example just trying to show a segment between two points A and B.
The points and the segment label are not shown in my Windows Internet Explorer.
Here is the sample file:
http://math-sculpture.com/eq-sh/IE-Cindy-test-segment-and-points-not-working.html
Everythings works fine in Windows Firefox and also on Mac Firefox / Safari.

However, when I remove the "labeled: true" from the geometry definition of the segment, 
it works:
http://math-sculpture.com/eq-sh/IE-Cindy-test-segment-and-points-working.html

In the same way as above, a statement "a.labeled = true;" in the "csinit" crashes the thing.

So, it seems that the labeling of segments has not been implemented correctly
causing Internet Explorer to not display the CindyJS construction correctly.

In short:
this is working:
   {name: "a", type: "Segment", color: [0.0, 0.0, 1.0], args: ["A", "B"], size:2,visible:true}
this is not working (label not shown, no points shown):
    {name: "a", type: "Segment", color: [0.0, 0.0, 1.0], args: ["A", "B"], size:2,visible:true, labeled: true}

It costed me quite some time to trace down this issue.
So, I'd be super-happy if you'd solve this issue soon
because exporting very simple constructions from Cinderella to CindyJS does not work in Internet Explorer because of this. And I am currently working a lot with this feature.

Thanks a lot in advance!
Oliver

**Comment by montaga** [comment]
Thank you for creating this example and filing this report.
In general, we did not want to put too much efforts in making IE work, since Edge seems to be an alternative closer to the standards. But in this case it seems to be easy: It further traces down to the fact that IE currently does not implement `Math.sign`, for which we can write an easy polyfill.

**Comment by montaga** [comment]
This should be fixed now by #699
In order to use it, you can either wait for the next release of CindyJS or include `https://cindyjs.org/dist/snapshot/Cindy.js` into your files
